vendor/modules.txt

```diff
@@ -928,6 +928,7 @@ github.com/hashicorp/go-azure-sdk/resource-manager/servicebus/2021-06-01-preview
 github.com/hashicorp/go-azure-sdk/resource-manager/servicebus/2021-06-01-preview/topics
 github.com/hashicorp/go-azure-sdk/resource-manager/servicebus/2021-06-01-preview/topicsauthorizationrule
 github.com/hashicorp/go-azure-sdk/resource-manager/servicebus/2022-01-01-preview/namespaces
```

Comment:
Looks like the older API Version is still used elsewhere in the codebase? When updating API versions you can use [this tool](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/internal/tools/update-api-version) to update all references across the provider - like so:

```
go build . && ./update-api-version -service="servicebus" -old-api-version="2022-01-01-preview" -new-api-version="2022-10-01-preview"
```

Comparing the result of that command verses the files changed in this PR, it appears there's still a reference to the older API Version in the State Migration:

```
	modified:   ../../services/servicebus/client/client.go
	modified:   ../../services/servicebus/internal.go
	modified:   ../../services/servicebus/migration/namespace_network_rule_set.go
	modified:   ../../services/servicebus/servicebus_namespace_authorization_rule_data_source.go
	modified:   ../../services/servicebus/servicebus_namespace_authorization_rule_resource.go
	modified:   ../../services/servicebus/servicebus_namespace_data_source.go
	modified:   ../../services/servicebus/servicebus_namespace_disaster_recovery_config_data_source.go
	modified:   ../../services/servicebus/servicebus_namespace_network_rule_set_resource.go
	modified:   ../../services/servicebus/servicebus_namespace_network_rule_set_resource_test.go
	modified:   ../../services/servicebus/servicebus_namespace_resource.go
	modified:   ../../services/servicebus/servicebus_namespace_resource_test.go
	modified:   ../../services/servicebus/servicebus_queue_authorization_rule_resource.go
	modified:   ../../services/servicebus/servicebus_queue_data_source.go
	modified:   ../../services/servicebus/servicebus_queue_resource.go
	modified:   ../../services/servicebus/servicebus_topic_authorization_rule_resource.go
	modified:   ../../services/servicebus/servicebus_topic_data_source.go
	modified:   ../../services/servicebus/servicebus_topic_resource.go
```

Mind fixing that one?